glossary.rst

```diff
@@ -0,0 +1,46 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This page contains definitions for Flux-specific terms and concepts mentioned
+throughout the various guides and docs on our sites.
+
+------------------------------------------------------------------------------
+
+**Flux Broker**
+  A distributed message broker daemon that provides communications services
+  within a Flux instance.
+
+**Flux Instance**
+  A group of flux-broker processes which are launched via any parallel launch
+  utility that supports PMI (Process Manager Interface).
+
+**Hostlist**
+  A format supported by existing HPC tools to create a compact list of hosts by
+  name, consisting of a common prefix, a number, and an optional suffix.
+
+**Jobspec**
+  A file which defines the resources, tasks, and other attributes of a single
+  program.
+
+**Jobtap Plugin**
+  A built-in or external plugin that can be loaded into the job manager broker
+  module.
+
+**KVS**
```

Comment:
Probably should start with "Key Value Store" to say what it means. And add what people use it for (users? developers? both?)

Reply:
Oh, yes, can't believe I forgot to spell out the acronym (when that's what we talked about why the Glossary page was necessary 🤦).

> And add what people use it for (users? developers? both?)

Good point. Will also ping this to the flux-core guys to get more explanation.